src/components/Sum.jsx

```diff
@@ -1 +1,12 @@
-// export function Sum() {}
+import React from 'react';
+
+export const Sum = ({ a, b }) => {
+  const content = `Sum of ${a} and ${b} is ${a + b}`;
+
+  return <p>{content}</p>;
+};
```

Comment:
I don't think there's a need to create a variable, as with this approach you don't win neither in readability nor in ergonomics